### DISCUSSION TITLE
Suddenly Failed ALL my Plots - Failed to open (invalid plots) - Need Help!

### DISCUSSION THREAD

**Discussion by DanbarArgentina** [Chia-Network/chia-blockchain#6406] [Support]
Just returned from office to home, and all my plots (135) are Invalid. I re-installed the chia GUI, but still the same situation. I never moved the plots (4 hours ago the 135 were OK), and now them all: "These plots are invalid, you might want to delete them.". 
Can somebody help me? Thanks! 
![1-2](https://user-images.githubusercontent.com/85322982/120737393-60b46000-c4c4-11eb-970a-23b919526912.jpg)
![2-2](https://user-images.githubusercontent.com/85322982/120737397-61e58d00-c4c4-11eb-8be6-263b314a7de7.jpg)

**Comment by tomatheus** [comment]
me too, please someone fix this

**Comment by XIII39** [comment]
Are you farming with the Chia software/GUI or with hpool? You can't do both.

**Reply by DanbarArgentina** [reply]
Both at same time: Plotting on chía and Farming on Hpool.

**Comment by cccat6** [comment]
Restart your machine then try if it fixed?

**Reply by DanbarArgentina** [reply]
Did it...but same situation.

**Reply by kevinwarnerlopez** [reply]
hola pudiste solucionar? estoy en la misma situacion

**Comment by kevinwarnerlopez** [comment]
estoy en la misma situacion por favor ayuda

**Comment by DanbarArgentina** [comment]
Estás usando hpool?

**Comment by kevinwarnerlopez** [comment]
No no estoy usando

**Comment by DanbarArgentina** [comment]
Reinstalar el cliente de chía por completo a la 1.1.7, y se debería
arreglar solo.

**Comment by matteoaugelli** [comment]
I solved it by deleting the database in `~/.chia/mainnet/db/` and restarting. After this operation you need sync again with mainnet, but at least it works again.

To be safe, make a backup of the entire folder **mainnet**

```
$ cp ~/.chia/mainnet{,.bk}
$ rm -Rf ~/.chia/mainnet/db/*
$ sudo reboot
```

**Comment by kevinwarnerlopez** [comment]
Lo que hice mirando videos fue borrar la carpeta de mainnet y ya me
aparecen los plots, pero ahora no me sincroniza saben porque puede ser esto?

**Reply by matteoaugelli** [reply]
> Lo que hice mirando videos fue borrar la carpeta de mainnet y ya me aparecen los plots, pero ahora no me sincroniza saben porque puede ser esto? El mar., 15 de junio de 2021 09:56, matteoaugelli ***@***.***> escribió:
> […](#)
> I solved it by deleting the database in ~/.chia/mainnet/db/ and restarting. After this operation you need sync again with mainnet, but at least it works again. To be safe, make a backup of the entire folder *mainnet* $ cp ~/.chia/mainnet{,.bk} $ rm -Rf ~/.chia/mainnet/db/* $ sudo reboot — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#6406 (comment)](https://github.com/Chia-Network/chia-blockchain/discussions/6406#discussioncomment-873142)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUPNHQAFRCK234DPMJDPLYDTS5EX7ANCNFSM46B5E5RQ> .

Not remove all mainet folder, only DB;

check logs in  `~/.chia/mainnet/log/debug.log `

**Comment by kevinwarnerlopez** [comment]
el problema es que ya borre la carpeta enterra, hay alguna solucion como
para que vuelva a sincronizar?


<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by DanbarArgentina** [comment]
No, debes reinstalar el Chía GUI y esperar q sincronice todo desde cero.

**Comment by kevinwarnerlopez** [comment]
Perfecto ya me funciona, muchas gracias

**Comment by BrandtH22** [comment]
@DanbarArgentina , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
